.gitignore

```diff
@@ -22,6 +22,7 @@ public/dist/
 uploads
 modules/users/client/img/profile/uploads
 config/env/local.js
+config/env/local-*.js
```

Comment:
@codydaig, apologies, you are correct because the grunt copy function needs the `local.example.js`